docs/messaging/sms/postman.md

```diff
@@ -0,0 +1,75 @@
+
+The collection is available here:
+
+* [Postman 2.1 Collection](../specs/ringcentral_sms_rest_apis.postman_collection.json)
+<!--* [Postman 2.1 Environment](../specs/ringcentral_environment.postman_environment.json)-->
+
+This document describes how to install and use the Postman 2.1 Collection.
+
+## Pre-Requisites
+
+This Postman spec is designed for following environment:
+
+* Current RingCentral MVP account located at: https://www.ringcentral.com. 
+* RingCentral app created at https://developers.ringcentral.com with Password-based or JWT auth flow enabled.
+
+## Using Postman
+
+Using Postman once you have your pre-requisites consists of a few steps:
+
+1. Importing the Postman Collection
+2. Creating/Configuring Your Postman Environment
+3. Authenticating and Authorizing the user.
+4. Making an API call
+
+
+### Importing the Postman Collection
+
+Use the following steps to import the Postman collection.
+
+1. In the upper left corner of the Postman application click the "Import" button.
+2. Click the "Import from Link" tab.
+3. Paste in the following URL where it says "Enter a URL and press import": [`https://raw.githubusercontent.com/ringcentral/ringcentral-api-docs/master/docs/messaging/sms/spec/ringcentral_sms_rest_apis.postman_collection.json`](../specs/ringcentral_sms_rest_apis.postman_collection.json)
+4. Click the "Continue" button
+
+## Configuring Your Postman Environment
+
+The Postman Collection uses environment variables for authentication and authorization. Fill out thes following for your environment:
+
+1. In Postman, create an environment by clicking the Gear icon for "Management Environments" in the upper right corner. This will bring up a list of existing environments.
+2. Click "Add" to create a new environment.
+3. Choose a name of your choice.
+4. Enter your enviroment variables as described below.
+5. Click the "Add" button to finish adding this environment.
+You can also use this [Postman 2.1 Environment](../specs/ringcentral_environment.postman_environment.json) export as reference. 
+
+<img class="img-fluid" width="100%" src="../../../img/postman_ringcentral_environment.png">
+
+| Variable | Description |
+|------|-------------|
+| **`RINGCENTRAL_SERVER_URL`** | RingCentral's API Url which would be https://platform.ringcentral.com for Production and https://platform.devtest.ringcentral.com for Sandbox|
+| **`RINGCENTRAL_CLIENT_ID`** | Application's Client ID|
+| **`RINGCENTRAL_CLIENT_SECRET`** | Application's Client Secret |
+| **`RINGCENTRAL_USERNAME`** | RingCentral username |
+| **`RINGCENTRAL_EXTENSION`** | RingCentral user's extension number |
+| **`RINGCENTRAL_PASSWORD`** | RingCentral user's password. Note, this needs to be using the RingCentral passsword system and not SSO for this Postman spec. |
+| **`RINGCENTRAL_JWT`** | RingCentral user's JWT token from User's Profile -> Credentials |
```

Comment:
Link to instructions if you can. Also make it clear that one either provides a username/password/extension or a JWT, not both.

Reply:
This is done now.